application/ui/widgets/eh-widgets-dir/eh-widgets/src/components/forms/BundleGroupForm/BundleGroupForm.js

```diff
@@ -314,6 +314,53 @@ const BundleGroupForm = ({
         }
     }
 
+    const replaceTextAreaToCKEditor =() => {
+        let formDescriptionTextArea = document.querySelector('#description');
+        if(formDescriptionTextArea && formDescriptionTextArea.style && formDescriptionTextArea.style.display !== 'none'){
+            ClassicEditor
+                .create( document.querySelector( '#description' ), {
+                    toolbar: {
+                        items: [
+                            'undo', 'redo',
+                            '|',
+                            'selectAll',
+                            '|',
+                            'heading',
+                            '|',
+                            'bold', 'italic',
+                            // 'underline', 'code', 'subscript', 'superscript',
```

Comment:
Do you disable these? I tried enabling locally and didn't see a change so I'm not quite sure what to make of it.